wiki/Skinning/en.md

```diff
@@ -1,22 +1,106 @@
 # Skinning
 
-Skinning is one of the fundamentals of all osu! game modes.
+Skinning is one of the fundamentals of all _osu!_ game modes.
 It enables users to derive from the original skinning elements to create their own!
 
-Over the years for osu!, skinning have changed dramatically in style.
-There are various ways that you can skin osu!.
+Over the years of _osu!_, skinning have changed dramatically in style.
+There are various ways that you can skin _osu!_
 
-Skins vary from for-fun to a central theme (i.e anime, or group of people) to those that are nigh impossible to play with ([such as this](//osu.ppy.sh/forum/viewtopic.php?f=15&t=45689)).
-Another kind of skin are pro skins, where they maximize play field visibility and minimize visual obtrusions (i.e. replacing combobursts and hitbursts with blank images).
+Skins vary from for-fun to a central theme (e.g. anime or group of people) to those that are nigh impossible to play with (e.g. [Konata Skin](https://osu.ppy.sh/forum/t/45689)).
```

Comment:
you shouldnt give examples of "bad" skins; it's pretty rude to the author of it ;;